Test_BaseModel.py

```diff
@@ -0,0 +1,16 @@
+import unittest
```

Comment:
For the price of adding a dependency, consider the simpler API of Pytest:
https://pytest.org

Reply:
Mihai, I'll start using pytest for new tests and incrementally convert old tests over. 
I started by changing test_BaseModel.py to use pytest and I haven't figured out how to get logging messages to be printed out. 'pytest -s' is of course needed but logging only gets captured to WARN level by default. Some recommendations on the web are to install pytest-logging module or to set logging.basicConfig(level=logging.DEBUG, stream=sys.stdout) in the test script. Neither work for me. Any suggestions?

Reply:
I have just successfully tried `--log-cli-level DEBUG`, as suggested in the docs:
https://docs.pytest.org/en/latest/logging.html